evalml/tests/component_tests/test_simple_imputer.py

```diff
@@ -487,11 +487,10 @@ def test_simple_imputer_woodwork_custom_overrides_returned_by_components(
     except ww.exceptions.TypeConversionError:
         return
 
-    impute_strategy_to_use = impute_strategy
-    if logical_type in [NaturalLanguage, Categorical]:
-        impute_strategy_to_use = "most_frequent"
+    if logical_type in [NaturalLanguage, Categorical, Boolean, BooleanNullable]:
+        impute_strategy = "most_frequent"
```

Comment:
Not a huge fan of how this was originally done - with `impute_strategy` iterating over a subset of the total `impute_strategy` available and changing it in the test.   But that's not your problem...we might want to think about rewriting this.